hooks/hookstage/bidderrequest.go

```diff
@@ -27,3 +27,17 @@ type BidderRequestPayload struct {
 	Request *openrtb_ext.RequestWrapper
 	Bidder  string
 }
+
+func (brp *BidderRequestPayload) GetBidderRequestPayload() *openrtb_ext.RequestWrapper {
+	return brp.Request
+}
+
+func (brp *BidderRequestPayload) SetBidderRequestPayload(br *openrtb_ext.RequestWrapper) {
+	brp.Request = br
+}
```

Comment:
Yes, we need a unified interface to get and set RequestWraper from payloads where we have it. We also need this interface to check data type in `handleModuleActivities`, we only process activities if input type has a BidderRequest in payload:

```go
payloadData, ok := any(&payload).(hookstage.PayloadBidderRequest)
	if !ok {
		return payload
	}
```
With this interface we don't need to know the exact type of Payload, we only need to know if it has a BidderRequest.

Reply:
Gotcha, I see the need for the interface. I got a little confused by the name of the interface when looking at the code in `handleModuleActivities`. I originally thought the name `PayloadBidderRequest` in that function was referring to a specific type of payload, `BidderRequestPayload`, rather than an interface. I suggest we change the name of this interface to something like `RequestMutator`, `RequestReaderWriter`, etc. to reduce confusion and to have the interface avoid constraining us to a specific underlying data type.